mangaki/mangaki/utils/data.py

```diff
@@ -48,11 +48,12 @@ def save_csv(self, suffix=''):
         works_path = os.path.join(settings.DATA_DIR, 'works{}.csv'.format(suffix))
         confirm = True
         if os.path.isfile(ratings_path) or os.path.isfile(works_path):
-            confirm = input('Already exists. Continue? [y/n] ') == 'y'
+            confirm = input('`{}` or `{}` already exists. Overwrite? [y/n] '
+                            .format(ratings_path, works_path)) == 'y'
         if confirm:
             with open(ratings_path, 'w', newline='') as csvfile:
                 data = csv.writer(csvfile, delimiter=',', quotechar='', quoting=csv.QUOTE_NONE)
-                for (encoded_user_id, encoded_work_id), rating in zip(self.anonymized.X, self.anonymized.y):
+                for (encoded_user_id, encoded_work_id), rating in zip(self.anonymized.X, self.anonymized.original_y):
```

Comment:
Sorry, may we call it `y_origin` or `y_original` or `y_text`? Because we do a lot of `y_pred`, etc.